src/ops.jl

```diff
@@ -189,7 +184,7 @@ trace(A::FixedMatrix{2,2}) = A[1,1] + A[2,2]
 trace(A::FixedMatrix{3,3}) = A[1,1] + A[2,2] + A[3,3]
 trace(A::FixedMatrix{4,4}) = A[1,1] + A[2,2] + A[3,3] + A[4,4]
 
-\{m,n,T}(mat::Mat{m,n,T}, v::Vec{n, T}) = inv(mat)*v
+\{m,n,T1,T2}(mat::Mat{m,n,T1}, v::Vec{n,T2}) = inv(mat)*v
```

Comment:
I know this isn't a part of this PR, but maybe we should improve this at some point. I feel that `\` should work with rank-deficient matrices and matching rank-deficient RHS vectors, and perhaps should work in the rectangular matrix scenario also (returning smallest L2 solution in one case, or the "closest" L2 solution in the other).